html/statbrowser.html

```diff
@@ -0,0 +1,864 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Stat Browser</title>
+<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<style>
+body {
+	font-family: Verdana, Geneva, Tahoma, sans-serif;
+	font-size: 12px !important;
+	margin: 5px !important;
+	padding: 2px !important;
+}
+a {
+	color: black;
+	text-decoration: none
+}
+a:hover, .dark a:hover {
+	text-decoration: underline;
+}
+.dark a{
+	color: #abc6ec;
+}
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  background-color: #333;
+}
+
+li {
+  float: left;
+}
+
+li a {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+li a:hover:not(.active) {
+  background-color: #111;
+}
+
+.active {
+  background-color: #4CAF50;
+}
+.button-container {
+  display: inline-flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-start;
+  min-width: 0;
+  min-height: 0;
+  white-space: pre-wrap;
+}
+.button {
+  background-color: #FFFFFF;
+  border-color: #FFFFFE;
+  color: black;
+  padding: 7px 14px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 12px;
+  margin: 2px 2px;
+  cursor: pointer;
+  border-radius: 10%;
+  transition-duration: 0.25s;
+  order: 3;
+}
+.dark button {
+  background-color: #252525;
+  border-color: #252524;
+  color: white;
+}
+.button.Theme {
+	position: fixed;
+	top: auto;
+	bottom: 0px;
+	right: 0px;
+	float: right;
+	order: 10000;
+}
+.button:hover {
+	border-color: #FFFFFE;
+    background-color: black;
+    color: white;
+}
+.dark button:hover {
+	border-color: #252524;
+    background-color: white;
+    color: #252525;
+}
+.button:active, .button.active {
+  text-decoration: underline;
+}
+.grid-container {
+  display: inline-flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  min-width: 0;
+  min-height: 0;
+  white-space: pre-wrap;
+}
+.grid-item {
+  color: black;
+  width: 150px;
+  font-size: 11px;
+  line-height: 24px;
+  text-align: left;
+  min-width: 0;
+  min-height: 0;
+  white-space: pre-wrap;
+}
+.link {
+  display: inline;
+  background: none;
+  border: none;
+  padding: 7px 14px;
+  color: black;
+  text-decoration: none;
+  cursor: pointer;
+  font-size: 13px;
+  margin: 2px 2px;
+}
+.dark .link {
+	color: #abc6ec;
+}
+.link:hover {
+  text-decoration: underline;
+}
+</style>
+<ul id="menu" class="button-container">
+</ul>
+</head>
+<body>
+<script>
+var decoder = decodeURIComponent || unescape;
+// browser passthrough code copy pasted here
+if(document.addEventListener && window.location) { // hey maybe some bozo is still using mega-outdated IE
+	let anti_spam = []; // wow I wish I could use e.repeat but IE is dumb and doesn't have it.
```

Comment:
Use of `let` here and elsewhere means this script breaks for Wine users on Linux, which is stuck on IE8.

Partial or full IE8 incompatibility is acceptable for UIs which are not essential to the game, but the verb panel is definitely essential to the game. It needs to basically work on IE8.